planning/behavior_velocity_planner/src/scene_module/occlusion_spot/occlusion_spot_utils.cpp

```diff
@@ -415,61 +416,45 @@ void filterCollisionByRoadType(
   }
 }
 
-void generateSidewalkPossibleCollisions(
+void generateDetectionAreaPossibleCollisions(
+  std::vector<Slice> & detection_area_polygons,
```

Comment:
`const` can add to `detection_area_polygons`

Reply:
[b067661](https://github.com/autowarefoundation/autoware.universe/pull/354/commits/b0676615e86c71d3083f35e6221c0be97147a3a9)